src/core/gosling.schema.ts

```diff
@@ -209,6 +209,25 @@ export interface CommonTrackDef extends CommonViewDef {
 export interface DataTrack extends CommonTrackDef {
     data: DataDeep;
 }
+/**
+ * A placeholder track. In contrast to other tracks, this track does not display any data. Instead it provides
+ * empty space for third party tools to display their data on top of.
+ */
+export interface DummyTrack extends CommonTrackDef {
```

Comment:
ended up having to extend CommonTrackDef because too many of the other attributes were being used in other parts of the spec preprocessing

Reply:
We would need to clarify in the documentation that properties are actually unused.